README.md

```diff
@@ -21,4 +21,26 @@ project for use in the Sunrise training (WIP)
 
 ### bulky goods
 
-* add a custom line item if there are more than 3 line items
\ No newline at end of file
+* add a custom line item if there are more than 3 line items
+
+
+## Links
+
+#### Handlebars
+To learn how to write Handlebars templates, please check the [Handlebars.js](http://handlebarsjs.com/) documentation. In particular, the sections about [Expressions](http://handlebarsjs.com/expressions.html), [Built-In Helpers](http://handlebarsjs.com/builtin_helpers.html) and [@data Variables](http://handlebarsjs.com/reference.html#data).
+
+Try it out: http://tryhandlebarsjs.com/
+
+#### Java 8
+A general guide on the highlights of Java 8 can be found in this [Java 8 Tutorial](http://winterbe.com/posts/2014/03/16/java-8-tutorial/). Lamdba Expressions, Streams and Optionals are especially useful for Sunrise.
+
+A bigger topic is `CompletionStage`, which is constantly used by Sunrise and Play 2.5. Read the [Guide to Java 8 CompletionStage](http://www.nurkiewicz.com/2013/05/java-8-definitive-guide-to.html) and take a look at the [CompletableFutureUtils.java](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/utils/CompletableFutureUtils.java) utils class provided by JVM SDK, which contains some useful methods used in Sunrise, such as `recoverWith`.
```

Comment:
I fixed the link in https://github.com/commercetools/commercetools-sunrise-java-training/pull/1/commits/01dfd367d620a3be9c6ac27cf119da348452d73b 

You could also link the Javadoc: http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/utils/CompletableFutureUtils.html which contains sometimes examples in the method details.